docs/make.jl

```diff
@@ -2,6 +2,32 @@
 #
 #
 
+if "--help" ∈ ARGS
+    println(
+        """
+docs/make.jl
+
+Render the `Manopt.jl` documentation with optional arguments
+
+Arguments
+* `--exclude-tutorials` - exclude the tutorials from the menu of Documenter,
+  this can be used if you do not have Quarto installed to still be able to render the docs
+  locally on this machine. This option should not be set on CI.
+* `--help`              - print this help and exit without rendering the documentation
+* `--prettyurls`        – toggle the prettyurls part to true (which is otherwise only true on CI)
+* `--quarto`            – run the Quarto notebooks from the `tutorials/` folder before generating the documentation
+  this has to be run locally at least once for the `tutorials/*.md` files to exist that are included in
+  the documentation (see `--exclude-tutorials`) for the alternative.
+  If they are generated once they are cached accordingly.
+  Then you can spare time in the rendering by not passing this argument.
+  If quarto is not run, some tutorials are generated as empty files, since they
+  are referenced from within the documentation. These are currently
+  `Optimize.md` and `ImplementOwnManifold.md`.
```

Comment:
Maybe let's delete the last sentence here because it's likely to be outdated soon.